internal/command/operator.go

```diff
@@ -115,6 +115,10 @@ Note: If --auto-registry-credentials and --registry-credentials-path are unset,
 				}
 				registryCredentials = string(registryCredentialsBytes)
 			}
+			// warn the user if no credentials are set by this point
+			if registryCredentials == "" {
+				fmt.Println("Note: no image pull credentials specified, the operator will be deployed without an image pull secret. If operator images are not present or accessible then the operator will be unable to start.")
```

Comment:
I think this should be sent to stderr instead (`fmt.Fprint(os.Stderr, ".."`)
Else it will becomes part of the manifests output to stdout when `--stdout` option is passed and makes the manifests invalid to apply (try `go run main.go operator deploy --stdout | kubectl apply --dry-run=client -f -` from this branch)

Reply:
Ahh, yes. I'll get that updated.

Reply:
Fixed in https://github.com/jetstack/jsctl/pull/40/commits/db62bde55b0427deeff1abc94f236e43d219e4fc